API/scanner/scanner-adapter-openapi-v1.0.yaml

```diff
@@ -0,0 +1,392 @@
+openapi: 3.0.0
+info:
+  title: Harbor Scanner Adapter API
+  description: |
+    ## Overview
+
+    The API that has to be implemented in order to plug an artifact scanner in [Harbor](https://goharbor.io) registry.
+
+    The [/scan](#operation/AcceptScanRequest) and [/scan/{scan_request_id}/report](#operation/GetScanReport) operations
+    do the actual scanning and provide a scan report which can be displayed in the Harbor web console.
```

Comment:
The [/scan](#operation/AcceptScanRequest) and [/scan/{scan_request_id}/report](#operation/GetScanReport) operations are responsible for the actual scanning and return a scan report viewable in the Harbor web console.

Reply:
The /scan and /scan/{scan_request_id}/report operations are responsible for the actual scanning and return a scan report that is visible in the Harbor web console.